packages/terra-form-select/src/terra-dev-site/test/form-select/DefaultValueZero.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Select from '../../../Select';
+
+const DefaultValueZero = () => (
+<Select defaultValue={0} id="defaultValueZero" >
```

Comment:
I forgot that our test scripts run separate from our lint scripts. Run `npm run lint` at the `terra-form-select` package level. It will output the same lint errors as our Travis Continuous Integration (CI) build that kicks off for each pull request: https://travis-ci.org/cerner/terra-core/jobs/502650623#L482. We should be good once we fix the lints and clear up the test example